### PR TITLE
Fix links in finding-your-way.md

### PR DESCRIPTION
**Pull request by ahstro**
Fix some broken links

### CODE REVIEW

File: finding-your-way.md [hunk 1]
```diff
@@ -198,7 +198,7 @@ Do you like mentoring people?
 [how-to-exercise-readmes]: /language-tracks/exercises/anatomy/readmes.md
 [discussions-docs]: https://github.com/exercism/discussions/issues?q=is%3Aissue+is%3Aopen+documentation+sort%3Aupdated-desc
 [docs-issues]: https://github.com/exercism/docs/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc
-[org-issues-dogs]: https://github.com/search?o=desc&p=1&q=org%3Aexercism+documentation+is%3Aissue&ref=searchresults&s=updated&state=open&type=Issues&utf8=%E2%9C%93
```
Comment (Insti) [inline finding-your-way.md:201]: 🐶 🐕 🐩 😆

**Review by Insti — APPROVED** [review]
Awesome, thanks :heart:

File: finding-your-way.md (hunk 1 above)
Comment (ahstro) [reply finding-your-way.md:201]: I wouldn't mind some issue dogs, would make bug fixing easier to cope with 🐶 😍

**Comment by rpottsoh** [comment]
Thanks @ahstro

**Pull request merged**